### ISSUE TITLE
How to change the width of the alert?

### ISSUE THREAD

**Issue by dosymbaev** [Codigami/CFAlertViewController#23]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/Codigami/CFAlertViewController/blob/develop/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/CFAlertViewController/)
* [x] I have searched for a similar issue in the [project](https://github.com/Codigami/CFAlertViewController/issues) and found none

### Issue Info

 Info                          | Value                               |
-------------------------------|-------------------------------------|
 Platform Name                 | e.g. ios
 Platform Version              | e.g. 8.0
 CFAlertViewController Version | e.g. 2.1.1
 Integration Method            | e.g. cocoapods / manually
 Xcode Version                 | e.g. Xcode 11.0
 Repro rate                    | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj       | e.g. does it happen with our demo project?
 Demo project link             | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.
